src/env.test.ts

```diff
@@ -0,0 +1,121 @@
+import env from '@src/env'
+import { vi } from 'vitest'
+import { viteEnv, windowElectronProcessEnv, processEnv } from '@src/env'
+
+describe('@src/env default export', () => {
+  it('should run the process.env workflow', () => {
+    // vite > node.js
+    const expected = {
+      NODE_ENV: 'test',
+      VITE_KC_API_WS_MODELING_URL: 'wss://api.dev.zoo.dev/ws/modeling/commands',
+      VITE_KITTYCAD_API_BASE_URL: 'https://api.dev.zoo.dev',
+      VITE_KC_SITE_BASE_URL: 'https://dev.zoo.dev',
+      VITE_KC_SITE_APP_URL: 'https://app.dev.zoo.dev',
+      VITE_KC_CONNECTION_TIMEOUT_MS: '5000',
+      VITE_KITTYCAD_API_TOKEN: 'redacted',
+      PROD: undefined,
+      TEST: 'true',
+      DEV: '1',
+      CI: undefined,
+    }
+    const actual = env()
+    expect(typeof actual.VITE_KITTYCAD_API_TOKEN).toBe('string')
+    //@ts-ignore I do not want this token in our logs for any reason.
+    actual.VITE_KITTYCAD_API_TOKEN = 'redacted'
+    expect(actual).toStrictEqual(expected)
+  })
+  describe('viteEnv', () => {
```

Comment:
Is this meant to be indented under the `env()` tests? This feels like a top-level describe block. Ditto on others in this file.

Reply:
I renamed the parent describe to  `describe('@src/env', () => {` and then there are 4 under it for the 4 functions to test in that file. 

Is that clearer?